extensions/amp-social-share/1.0/social-share.js

```diff
@@ -160,10 +168,18 @@ function handleActivation(finalEndpoint, target) {
         `Could not complete system share.  Navigator unavailable. ${NAME}`
       );
     }
-  } else if (protocol === 'sms') {
+  } else if (protocol === 'sms' || protocol === 'mailto') {
+    if (
```

Comment:
Thoughts on pulling this condition into a doc level boolean function?

Reply:
Thanks, that's a good idea.  I've re-factored the code with this change in the newest version.